python/grass/gunittest/main.py

```diff
@@ -138,8 +138,9 @@ def get_config(start_directory, config_file):
 
     If file is explicitly specified, it must exist.
 
-    Raises OSError if file is not accessible, e.g., if it exists,
-    but there is an issue with permissions.
+    :raises OSError: if file is not accessible, e.g., if it exists,
+        but there is an issue with permissions.
+    :raises ValueError: If both start_directory and config_file are not set.
```

Comment:
"If neither ...nor... are set" sounds better to me

Reply:
Should be ready to go now